viewer/packages/tools/src/Measurement/MeasurementLabel.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { Cognite3DViewer } from '@reveal/core';
+import * as THREE from 'three';
+import { HtmlOverlayTool, HtmlOverlayToolOptions } from '../HtmlOverlay/HtmlOverlayTool';
+import labelCSS from './styles/Label.css';
+
+export class MeasurementLabel {
```

Comment:
This looks like it's only holding a single label, and using a HtmlOverlayTool to hold the single label. That it's not a good use of overlay-tool.

Or; does it hold several labels? If yes, then I have the following comments:
- add() is an ok name, but I dont like that it changes internal state (this._labelElement). I dont see the need for this.
- remove() is really removeLast(). Rather, make add() return the label and let remove() accept label to be removed. Caller should keep track of the instance to remove
- Rename class to `MeasurementLabels`